recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -211,117 +124,21 @@ this.PreferenceExperiments = {
       preferenceBranchType,
     };
 
-    const prevPrefType = Services.prefs.getPrefType(preferenceName);
-    const givenPrefType = PREFERENCE_TYPE_MAP[preferenceType];
+    // Validation will throw
```

Comment:
"Validation will throw." is a complete sentence. :-) I often like to use a colon on comments like this, if you want to emphasize that they apply to the following thing (rather than the preceding). Not sure if that's the case here…